doc/releases/changelog-0.12.0.md

```diff
@@ -258,112 +322,120 @@
+  `ppr_to_ppm` functionality now handles direct decomposition of PPRs into PPMs.
   [(#1688)](https://github.com/PennyLaneAI/catalyst/pull/1688)
 
-* The version of JAX used by Catalyst is updated to 0.6.0.
+* The version of JAX used by Catalyst has been updated to v0.6.0.
   [(#1652)](https://github.com/PennyLaneAI/catalyst/pull/1652)
   [(#1729)](https://github.com/PennyLaneAI/catalyst/pull/1729)
 
   Several internal changes were made for this update.
-    - LAPACK kernels are updated to adhere to the new JAX lowering rules for external functions.
-    [(#1685)](https://github.com/PennyLaneAI/catalyst/pull/1685)
+  - LAPACK kernels are updated to adhere to the new JAX lowering rules for external functions.
+  [(#1685)](https://github.com/PennyLaneAI/catalyst/pull/1685)
 
-    - The trace stack is removed and replaced with a tracing context manager.
-    [(#1662)](https://github.com/PennyLaneAI/catalyst/pull/1662)
+  - The trace stack is removed and replaced with a tracing context manager.
+  [(#1662)](https://github.com/PennyLaneAI/catalyst/pull/1662)
 
-    - A new `debug_info` argument is added to `Jaxpr`, the `make_jaxpr`
-    functions, and `jax.extend.linear_util.wrap_init`.
-    [(#1670)](https://github.com/PennyLaneAI/catalyst/pull/1670)
-    [(#1671)](https://github.com/PennyLaneAI/catalyst/pull/1671)
-    [(#1681)](https://github.com/PennyLaneAI/catalyst/pull/1681)
+  - A new `debug_info` argument is added to `Jaxpr`, the `make_jaxpr` functions, and 
+  `jax.extend.linear_util.wrap_init`.
+  [(#1670)](https://github.com/PennyLaneAI/catalyst/pull/1670)
+  [(#1671)](https://github.com/PennyLaneAI/catalyst/pull/1671)
+  [(#1681)](https://github.com/PennyLaneAI/catalyst/pull/1681)
 
-* (Compiler developers only) The version of LLVM, mlir-hlo and Enzyme used by Catalyst is
-  updated to track those in jax 0.6.0.
+* The version of LLVM, mlir-hlo, and Enzyme used by Catalyst has been updated to track those in JAX 
+  v0.6.0.
   [(#1752)](https://github.com/PennyLaneAI/catalyst/pull/1752)
 
-  The LLVM version is updated to [commit 179d30f8c3fddd3c85056fd2b8e877a4a8513158](https://github.com/llvm/llvm-project/tree/179d30f8c3fddd3c85056fd2b8e877a4a8513158).
-  The mlir-hlo version is updated to [commit 617a9361d186199480c080c9e8c474a5e30c22d1](https://github.com/tensorflow/mlir-hlo/tree/617a9361d186199480c080c9e8c474a5e30c22d1).
-  The Enzyme version is updated to [v0.0.180](https://github.com/EnzymeAD/Enzyme/releases/tag/v0.0.180).
+  The LLVM version has been updated to 
+  [commit a8513158](https://github.com/llvm/llvm-project/tree/179d30f8c3fddd3c85056fd2b8e877a4a8513158).
+  The mlir-hlo version has been updated to 
+  [commit e30c22d1](https://github.com/tensorflow/mlir-hlo/tree/617a9361d186199480c080c9e8c474a5e30c22d1).
+  The Enzyme version has been updated to 
+  [v0.0.180](https://github.com/EnzymeAD/Enzyme/releases/tag/v0.0.180).
 
 * The clang-format and clang-tidy versions used by Catalyst have been updated to v20.
```

Comment:
This is not a breaking change, at best this could be classified as an internal change (but we can probably omit it entirely).